sampleproject/components/greeting.py

```diff
@@ -1,13 +1,29 @@
+from typing import Any, Dict
+
+from django.http import HttpResponse
+
 from django_components import component
 
 
 @component.register("greeting")
-class greeting(component.Component):
-    def get_context_data(self, greet, *args, **kwargs):
-        return {"greet": greet}
+class Greeting(component.Component):
+    def get(self, request, *args, **kwargs):
+        context = self.get_context_data(request, *args, **kwargs)
+        slots = [("message", "Hello, world!", None)]
+        self.fill_slots(slots)
+        rendered_component = self.render(context)
+        return HttpResponse(rendered_component)
```

Comment:
Hmm. This is the core new API right? I wonder if it would make more sense for self.render to get a new slots keyword arg, where you send in a dict with "name of slot" and "contents of slot". I think it might make it easier to understand what's going on... What do you think?

Reply:
I updated this and looks much better when passing a dictionary with the slots data!